### ISSUE TITLE
Auto integrate with behat fail aid

### ISSUE THREAD

**Issue by forceedge01** [forceedge01/sql-data-mods#33]
Last queries to be added to state.

**Comment by forceedge01** [comment]
Due to the overwhelming sql that is executed in the background and not to bloat the failure screen it may be wise to consider selective state additions i.e in the behat.yml the extension should be configurable like so:

```yml
SQLExtension:
  FailAid:
    Output:
      select: (bool)
      inserts: (bool)
      update: (bool)
      delete: (bool)
```

Default to insert to true only.

**Comment by forceedge01** [comment]
Released in v4.18.0
